plugins/catalog/src/components/EntityLayout/EntityLayout.tsx

```diff
@@ -119,6 +119,7 @@ function EntityLabels(props: { entity: Entity }) {
       {ownedByRelations.length > 0 && (
         <HeaderLabel
           label="Owner"
+          contentTypograpyRootComponent="p"
```

Comment:
The actual fix is to use `p` instead of `span` for the owner links.
Specifying this here in this implementation only does not change the implementation of this component elsewhere (where `span` is still used)